src/main/kotlin/io/snyk/plugin/services/SnykApplicationSettingsStateService.kt

```diff
@@ -21,18 +22,25 @@ class SnykApplicationSettingsStateService : PersistentStateComponent<SnykApplica
     var organization: String? = null
     var ignoreUnknownCA = false
     var cliVersion: String? = null
-    var lastCheckDate: Date? = null
+
     var cliScanEnable: Boolean = true
     var snykCodeSecurityIssuesScanEnable: Boolean = true
     var snykCodeQualityIssuesScanEnable: Boolean = false
     var sastOnServerEnabled = false
     var usageAnalyticsEnabled = true
-    var pluginFirstRun = true
+
     var lowSeverityEnabled = true
     var mediumSeverityEnabled = true
     var highSeverityEnabled = true
     var criticalSeverityEnabled = true
 
+    var lastCheckDate: Date? = null
+    var pluginFirstRun = true
+    // Instant could not be used here due to serialisation Exception
+    var pluginFirstInstallTime: Date = Date.from(Instant.now())
+    var lastTimeFeedbackRequestShown: Date = Date(0) // January 1, 1970, 00:00:00 GMT
+    var doNotShowFeedbackRequest = false
```

Comment:
Our other boolean config parameters are `<something>Enabled`, so having variable `showFeedbackRequest = true` will be consistent with other config params and conditions checks will be simpler to read i.e.
```
if (!settings.doNotShowFeedbackRequest...
...
if (settings.showFeedbackRequest...
```
wdyt?